src/drive/web/modules/views/Public/index.jsx

```diff
@@ -188,7 +188,7 @@ const PublicFolderView = ({
   }, [setIsFabDisplayed, isMobile, hasWritePermissions])
 
   const showNewBreadcrumbFlag = flag(
-    'drive.breadcrumb.showCompleteBreadcrumbOnPublicPage'
+    'drive.breadcrumb.show-complete-breadcrumb-on-public-page'
```

Comment:
Same this is already used. Let's keep this one and remove it pretty soon

Reply:
Suggestion: why not specifying an end date for each feature flags? (or at least, a date that flag is outdated)